design-toolkit/components/src/components/drawer/drawer.stories.tsx

```diff
@@ -16,12 +16,16 @@ import type { Meta, StoryObj } from '@storybook/react';
 import type { ComponentProps } from 'react';
 import type { DrawerMenuProps, DrawerProps } from './types';
 
+// TODO: more work is needed to clean up the types for easier adoption of Storybook patterns
+// this story has a mix of controls from different components
```

Comment:
any future ticket for this todo or plan of attack?

Reply:
Added PATH-826.